ast/ast.go

```diff
@@ -101,16 +101,41 @@ func (Insert) isStatement()             {}
 func (Delete) isStatement()             {}
 func (Update) isStatement()             {}
 
-// QueryExpr represents set operator operands.
+// QueryExpr represents query expression, which can be body of QueryStatement or subqueries.
+// Select and FromQuery are leaf QueryExpr and others wrap other QueryExpr.
 type QueryExpr interface {
 	Node
 	isQueryExpr()
 }
 
 func (Select) isQueryExpr()        {}
+func (Query) isQueryExpr()         {}
+func (FromQuery) isQueryExpr()     {}
 func (SubQuery) isQueryExpr()      {}
 func (CompoundQuery) isQueryExpr() {}
 
+/*
```

Comment:
Could you remove this comment out?

Reply:
Fixed https://github.com/cloudspannerecosystem/memefish/pull/201/commits/c8a74d98efeee52e1014fa49b14793d4c0fdadcc